to-do-list/src/List.js

```diff
@@ -0,0 +1,26 @@
+import "./List.css";
+
+const ToDoList = () => {
+  return (
+    <div className="to-do-list">
+      <header>To Do List</header>
+      <input id="input-item" type="text"></input>
+      <button onClick={e => addItem(e.view.document)}>Add To-Do</button>
+      <div>
+        <ul id="list"></ul>
+      </div>
```

Comment:
To use the React features, you can convert this piece of code into sections. For example, the above lines can be similarly written as:
`<Header />
     <ToDoList list={list} />
      <Item addItem={addItem}/>`